zerver/migrations/0240_usermessage_migrate_bigint_id_into_id.py

```diff
@@ -15,16 +15,25 @@ class Migration(migrations.Migration):
             DROP TRIGGER zerver_usermessage_bigint_id_to_id_trigger ON zerver_usermessage;
             DROP FUNCTION zerver_usermessage_bigint_id_to_id_trigger_function();
 
-            ALTER TABLE zerver_usermessage ALTER COLUMN bigint_id SET NOT NULL;
-            ALTER TABLE zerver_usermessage DROP CONSTRAINT zerver_usermessage_pkey;
-            DROP SEQUENCE zerver_usermessage_id_seq CASCADE;
-            ALTER TABLE zerver_usermessage RENAME COLUMN id to id_old;
-            ALTER TABLE zerver_usermessage RENAME COLUMN bigint_id to id;
-            ALTER TABLE zerver_usermessage ADD CONSTRAINT zerver_usermessage_pkey PRIMARY KEY USING INDEX zerver_usermessage_bigint_id_idx;
-            CREATE SEQUENCE zerver_usermessage_id_seq;
-            SELECT SETVAL('zerver_usermessage_id_seq', (SELECT MAX(id)+1 FROM zerver_usermessage));
-            ALTER TABLE zerver_usermessage ALTER COLUMN id SET DEFAULT NEXTVAL('zerver_usermessage_id_seq');
-            ALTER TABLE zerver_usermessage ALTER COLUMN id_old DROP NOT NULL;
+            ALTER TABLE zerver_usermessage
+                ALTER COLUMN bigint_id SET NOT NULL,
+                DROP CONSTRAINT zerver_usermessage_pkey,
+                ALTER COLUMN id DROP IDENTITY IF EXISTS,
+                ALTER COLUMN id DROP NOT NULL,
+                ALTER COLUMN id DROP DEFAULT;
+            ALTER TABLE zerver_usermessage RENAME COLUMN id TO id_old;
+            ALTER TABLE zerver_usermessage RENAME COLUMN bigint_id TO id;
+            DROP SEQUENCE IF EXISTS zerver_usermessage_id_seq;
+            ALTER TABLE zerver_usermessage
+                ADD CONSTRAINT zerver_usermessage_pkey PRIMARY KEY USING INDEX zerver_usermessage_bigint_id_idx,
+                ALTER COLUMN id ADD GENERATED BY DEFAULT AS IDENTITY;
+            SELECT setval(
+                'zerver_usermessage_id_seq',
+                GREATEST(
+                    (SELECT max(id) FROM zerver_usermessage),
+                    (SELECT max(id) FROM zerver_archivedusermessage)
+                )
```

Comment:
I guess this commit message should document why this migration needed to be edited.